testament/tester.nim

```diff
@@ -500,10 +500,12 @@ proc main() =
     p.next
     processCategory(r, cat, p.cmdLineRest.string)
   of "r", "run":
-    let (dir, file) = splitPath(p.key.string)
```

Comment:
Please don't touch testament, it is unrelated to your ``unittest`` changes.

Reply:
Testament change moved to its own PR: https://github.com/nim-lang/Nim/pull/9971